backend/private-graph/graph/schema.graphqls

```diff
@@ -2116,7 +2124,13 @@ type Mutation {
 		is_opt_out: Boolean!
 		project_id: Int
 	): Boolean!
-	editService(id: ID!, project_id: ID!, github_repo_path: String): Service
+	editServiceGithubSettings(
```

Comment:
Renamed to github settings related since this does some other logic around setting/reseting null values